cli/src/build.rs

```diff
@@ -183,7 +201,13 @@ pub fn build_project(
             }
         }
 
-        print_binding_message(platforms)?;
+        // Write selected platforms for create command
+        for p in &selected_platforms {
+            config.target_platforms.push(p.to_string())
+        }
+        write_config(&config_path, &config)?;
```

Comment:
when I execute `mopro build` for iOS and `mopro create` for iOS
and execute again `mopro build` for Android and `mopro create`
The `target_platforms` is overwritten
it should be `target_platforms = ["ios","android"]` but it shows `target_platforms = ["android"]`
(so I cannot create for react-native even if I have iOS bindings)

try to read the config first and push the config if needed?